lib/records/fhir.rb

```diff
@@ -37,6 +37,9 @@ def self.basic_info(entity, fhir_record, end_time = Time.now)
         end
```

Comment:
There should be similar changes in the FHIR DSTU2 exporter as well. Take a look at the CCDA exporter as well but we've let that fall significantly behind already so updating that isn't critical.